app/subscriber/src/features/my-reports/admin/components/ReportSectionMediaAnalytics.tsx

```diff
@@ -144,6 +152,112 @@ export const ReportSectionMediaAnalytics = React.forwardRef<
           </Row>
         </Col>
       </Show>
+      <Show visible={!values.template?.chartTemplates.length}>
```

Comment:
We can now add/remove specific chart types again.  Didn't realize I had removed this.